### ISSUE TITLE
DRY

### ISSUE THREAD

**Issue by Oksana13** [naveenkumarr05/weather-app#14]
If we'd have 10 tabs, with this approach we'd need to copy/paste all this state changes a lot of times, which is definitely not DRY

https://github.com/naveenkumarr05/weather-app/blob/89bbedf3d4fd693a7f39c6db777a1ad077478127/src/components/DayForecastView.vue#L31

**Comment by naveenkumarr05** [comment]
yeah Correct if we have more tabs then we should go for other way like v-tabs
